src/Assets/States/NC/WhiteLabels/LancAssets/lancLink.ts

```diff
@@ -0,0 +1,10 @@
+const LancLinks = [
```

Comment:
This is more standard way to do constants.
```suggestion
const LANC_LINKS= [
```